doc install: add the install procedure for Debian 13 (#2507)
groonga/groonga: doc/source/install/debian.rst

  % sudo apt install -y -V groonga-normalizer-mysql

bookworm
--------

.. versionadded:: 13.0.2

> bookworm's install procedure is same trixie's install procedure.
> Please refer :ref:`trixie`.

Reviewer: ```suggestion
On bookworm, follow the same installation procedure as trixie. See/Please refer :ref:`trixie`.
```
